tasks/Module 2/task_1.py

```diff
@@ -0,0 +1,77 @@
+"""Task 1 and 2 of 2nd Module."""
+
+
+# Globals
+# TODO: move globals to separate setting module?
```

Comment:
Indeed it may be a good idea to move it to separate module if your program grows, but let's leave it here for now :)